### ISSUE TITLE
Formatting in logging functions

### ISSUE THREAD

**Issue by antnieszka** [DataMedSci/beprof#124]
According to this: https://landscape.io/github/DataMedSci/beprof/17/messages/smell
> Used when a logging statement has a call form of logging.<logging method>(format_string.format(format_args...)). Such calls should use % formatting instead, but leave interpolation to the logging function by passing the parameters as arguments. For more, see PEP-282 (https://www.python.org/dev/peps/pep-0282/)

We should - ` Use % formatting in logging functions and pass the % parameters as arguments `.

**Comment by antnieszka** [comment]
https://landscape.io/github/ant6/beprof/46
```
+ 4% compared to #45
SCORE INCREASED

0 new problems.
11 problems were fixed. 
```

**Comment by antnieszka** [comment]
Closed in #123
